src/darker/__main__.py

```diff
@@ -4,7 +4,7 @@
 import sys
 from difflib import unified_diff
 from pathlib import Path
-from typing import Generator, Iterable, List, Tuple
+from typing import Generator, Iterable, List, Optional, Tuple
```

Comment:
This import can be omitted if the shorter form `root: Path = None` is used below in `print_diff()` signature.
```suggestion
from typing import Generator, Iterable, List, Tuple
```